installing/install_config/customizations.adoc

```diff
@@ -5,91 +5,215 @@ include::modules/common-attributes.adoc[]
+|cluster
+|Configures how specific xref:../../openshift_images/image-configuration.adoc#image-configuration[image registries] should be treated (allowed, disallowed, insecure, CA details).
+|Global
+
 
 |Infrastructure
-|In {product-title} {product-version}, you cannot customize the Infrastructure resource.
+|config.openshift.io
+|cluster
+|Configuration details allowing the cluster to interact with its cloud provider.
+|Global
+
 
 |Ingress
-|To enable external access to {product-title} cluster services, configure the
-xref:../../networking/ingress-operator.adoc#configuring-ingress[Ingress Operator].
+|config.openshift.io
+|cluster
+|Configuration details related to routing such as the default domain for routes.
+|Global
+
 
 |Network
-|You cannot modify your cluster networking after installation. If you must
-customize your network, you
-xref:../../installing/installing_aws/installing-aws-network-customizations.adoc#installing-aws-network-customizations[customize networking during installation].
+|config.openshift.io
+|cluster
+|Configures the cluster network such as ip pools for services.
+|Global
+
+
+|Oauth
+|config.openshift.io
+|cluster
+|Configures identity providers and other behavior related to xref:../../authentication/configuring-internal-oauth.adoc#configuring-internal-oauth[internal OAuth server] flows.
+|Global
 
-|OAuth
-|Configure the
-xref:../../authentication/configuring-internal-oauth.adoc#configuring-internal-oauth[internal OAuth server].
 
 |Project
-|Configure xref:../../applications/projects/configuring-project-creation.adoc#configuring-project-creation[how projects are created].
+|config.openshift.io
+|cluster
+|Configures xref:../../applications/projects/configuring-project-creation.adoc#configuring-project-creation[how projects are created] including the project template.
+|Global
+
+
+|Proxy
+|config.openshift.io
+|cluster
+|Defines proxies to be used by components needing external network access.  Note: not all components currently consume this value.
+|Global
 
-|Samples
-|Configure the
-xref:../../openshift_images/configuring-samples-operator.adoc#configuring-samples-operator[samples Operator]
-to start using images.
 
 |Scheduler
-|To determine the default placement of pods on nodes, configure the
-xref:../../nodes/scheduling/nodes-scheduler-default.adoc#nodes-scheduler-default[default scheduler to control pod placement].
+|config.openshift.io
+|cluster
+|Configures xref:../../nodes/scheduling/nodes-scheduler-default.adoc#nodes-scheduler-default[scheduler] behavior such as policies and default nodeselectors.
+|Global
+
+|===
+
+[id="cluster-operator-configuration-resources_{context}"]
+== Cluster operator configuration resources
+
+[cols="2a,5a",options="header"]
+[cols="2a,2a,8a,2a,2a",options="header"]
+|===
+|Name
+|Group
+|Instance
+|Description
+|Namespace
+
+|Alertmanager
+|monitoring.coreos.com
+|main
+|Controls the alertmanager deployment parameters
+|openshift-monitoring
+
+
+|Authentication
```

Comment:
We do not wish to document this for cluster-admins.  This is not an area where they are allowed to alter configuration and status is reported back through our clusteroperator, so they do not need to view this resource for status either.

Reply:
will remove